Libraries/LibWeb/SRI/SRI.cpp

```diff
@@ -90,7 +90,7 @@ ErrorOr<Vector<Metadata>> parse_metadata(StringView metadata)
         auto algorithm = hash_expr_token_list[0];
 
         // 6. If hash-expr-token-list[1] exists, set base64-value to hash-expr-token-list[1].
-        if (hash_expr_token_list.size() >= 1)
+        if (hash_expr_token_list.size() > 1)
```

Comment:
Good catch! Can you add a test that exercises this issue?

Reply:
Done !